src/script/interpreter.cpp

```diff
@@ -683,10 +682,8 @@ bool EvalScript(std::vector<valtype> &stack, const CScript &script,
                         valtype &vch1 = stacktop(-2);
                         valtype &vch2 = stacktop(-1);
 
-                        int depthLonger =
-                            vch1.size() > vch2.size() ? 2 : 1;
-                        int depthShorter =
-                            vch1.size() > vch2.size() ? 1 : 2;
+                        int depthLonger = vch1.size() > vch2.size() ? 2 : 1;
```

Comment:
Can you run the formatter over this file before doing this commit or don't patch add without this stuff?